js/codemeta_generation.js

```diff
@@ -7,8 +7,21 @@
 
 "use strict";
 
+const CODEMETA_CONTEXT_URL = 'https://doi.org/10.5063/schema/codemeta-2.0';
+const CONTEXTS = {
+    [CODEMETA_CONTEXT_URL]: 'https://raw.githubusercontent.com/codemeta/codemeta/2.0/codemeta.jsonld'
+}
```

Comment:
Ditto, could you add these files to this repository instead of sending external requests? (even though both are going to be served by GitHub in practice, raw.githubusercontent.com is probably not on the same infrastructure as codemeta.github.io)

Reply:
Edit to my previous message: it does not seem necessary (I previously thought it was also making calls to schema.org).

Reply:
Indeed, we don't use schemas thanks to the custom validation.